e2e/smartvol_ops_test.go

```diff
@@ -28,6 +28,12 @@ func brickSizeTest(brickpath string, min uint64, max uint64) error {
 	return fmt.Errorf("Unable to get size info of Brick(%s)", brickpath)
 }
 
+func checkZeroLvs(r *require.Assertions) {
+	r.Equal(numberOfLvs("vg-dev-gluster_loop1"), 0)
```

Comment:
instead of this can we do it in for loop

```
for i:=1;i<3;i++{
    r.Equal(numberOfLvs(fmt.Sprintf("vg-dev-gluster_loop%d",i), 0)
}
```
and move `vg-dev-gluster_loop` to global var 

by changing the value of `i`, we can have any number of `numberOfLvs`